2 backend

EyadAbdelfatah opened the pull request:
made views that can sync from nfc-backend

EyadAbdelfatah committed:
d

benjaminbernadskiy committed:
Create .gitignore

EyadAbdelfatah committed:
hv

benjaminbernadskiy committed:
Update settings.py

benjaminbernadskiy committed:
Update settings.py

benjaminbernadskiy committed:
Update settings.py

benjaminbernadskiy committed:
trying cors

EyadAbdelfatah committed:
n

EyadAbdelfatah committed:
j

EyadAbdelfatah committed:
probably works

EyadAbdelfatah committed:
nn

EyadAbdelfatah committed:
me when i have duplicate serializers

Shatterdest commented on backend/scans/models.py:
```diff
@@ -0,0 +1,69 @@
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    name = models.CharField(max_length=100)
+    homeroom = models.CharField(max_length=4)
+    gradYear = models.IntegerField()
```
grad_year

Shatterdest commented on backend/scans/models.py:
```diff
@@ -0,0 +1,69 @@
+    homeroom = models.CharField(max_length=4)
+    gradYear = models.IntegerField()
+    email = models.EmailField(null=True, blank=True)
+    osis = models.CharField(max_length=9, null=True, blank=True)
```
well probably use one of either email or osis or caass in prod not all three u can keep this for now

Shatterdest commented on backend/scans/models.py:
```diff
@@ -0,0 +1,69 @@
+        return f"{self.name}"
+
+
+class ScanInstance(models.Model):
```
what's the point of storing scaninstances whne u can js take it from nfc backend --- defeats the purpose of our multi api system

Shatterdest commented on backend/scans/models.py:
```diff
@@ -0,0 +1,69 @@
+        return f"{self.student.name} "
+
+
+class Event(models.Model):
```
this too

Shatterdest commented on backend/scans/models.py:
```diff
@@ -0,0 +1,69 @@
+        return f"{self.name}"
+
+
+class Device(models.Model):
```
sdf

Shatterdest commented on backend/scans/models.py:
```diff
@@ -0,0 +1,69 @@
+        return f"{self.id}"
+
+
+class MicroService(models.Model):
```
sadfadsf

Shatterdest commented on backend/scans/views.py:
```diff
@@ -0,0 +1,124 @@
+from rest_framework.decorators import action
+
+
+BASE_URL = settings.NFC_BASE_URL
```
env but thats for later

Shatterdest reviewed (changes requested):
y did bro clone nfc-backend

EyadAbdelfatah committed:
a

EyadAbdelfatah committed:
a

RichardM1337 committed:
Create 0002_activity_remove_event_microservice_and_more.py

forgotten migration possibly

benjaminbernadskiy committed:
Update settings.py

benjaminbernadskiy commented:
<img width="168" height="52" alt="image" src="https://github.com/user-attachments/assets/bdc92837-04a9-4edc-a40a-aabb1d16c57f" />
gang r we srs

RichardM1337 committed:
DRF-SPECTACULAR

hanks kipnis for the rec
ive pusehd this straight to baceknd repo because it doesnt seem to require extra branch testing.
this package displays endpoints for documetnation purpises
